<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPWPage</class>
 <widget class="QWidget" name="ForgetPWPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>624</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>824</width>
    <height>624</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>824</width>
    <height>624</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>801</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_SignUpBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>390</width>
        <height>420</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>390</width>
        <height>450</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <widget class="QPushButton" name="pushButton_ResetPW">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton_ResetPW:enabled{
	border-width:0;
	background-color:#0a66c2;
	border-radius:20px;
	color:#ffffff;
}
QPushButton#pushButton_ResetPW:hover{
	border-width:0;
	background-color:#004182;
	border-radius:20px;
	color:#ffffff;
}
QPushButton#pushButton_ResetPW:pressed{
	border-width:0;
	background-color:#004182;
	border-radius:20px;
	color:#ffffff;
}
QPushButton#pushButton_ResetPW:focus{
	border-width:0;
	background-color:#004182;
	border-radius:20px;
	color:#ffffff;
}</string>
       </property>
       <property name="text">
        <string>重设密码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_fMail">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border: 1px solid gray;
	background-color:transparent;
	padding:0 0 0 8px;
	border-radius:8px;
	color:#666666;
}
QLineEdit:focus{
	color:#000000;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>邮箱</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>160</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>忘记密码？</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>350</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>快速重设密码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_fUserName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border: 1px solid gray;
	background-color:transparent;
	padding:0 0 0 8px;
	border-radius:8px;
	color:#666666;
}
QLineEdit:focus{
	color:#000000;
}</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>想起密码了？</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Back">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>400</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton_Back:enabled{
	border-width:0;
	background-color:transparent;
	border-radius:20px;
	color:#0a66c2;
}
QPushButton#pushButton_Back:hover{
	border-width:0;
	background-color:#d0e8ff;
	border-radius:15px;
	color:#0a66c2;
}
QPushButton#pushButton_Back:pressed{
	border-width:0;
	background-color:#d0e8ff;
	border-radius:15px;
	color:#0a66c2;
}</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_fNewPassword">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border: 1px solid gray;
	background-color:transparent;
	padding:0 0 0 8px;
	border-radius:8px;
	color:#666666;
}
QLineEdit:focus{
	color:#000000;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>新密码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_fRepeatPassword">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	border: 1px solid gray;
	background-color:transparent;
	padding:0 0 0 8px;
	border-radius:8px;
	color:#666666;
}
QLineEdit:focus{
	color:#000000;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>重复新密码</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
